{{cookiecutter.project_name}}/{{cookiecutter.project_name}}/celery.py

```diff
@@ -0,0 +1,47 @@
+import logging
+import os
+
+from celery import Celery as BaseCelery
+from celery.utils.time import timezone as celery_timezone
+from django.conf import settings
+from django.utils import timezone
+
+# set the default Django settings module for the 'celery' program.
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "{{ cookiecutter.project_name }}.settings.local")
+
+logger = logging.getLogger(__name__)
+
+
+class Celery(BaseCelery):
+    @property
+    def timezone(self):
+        return celery_timezone.get_timezone(settings.TIME_ZONE)
+
+    def now(self):
+        return timezone.localtime()
+
+
+app = Celery("{{cookiecutter.project_name}}")
+# Using a string here means the worker doesn't have to serialize
+# the configuration object to child processes.
+# - namespace='CELERY' means all celery-related configuration keys
+#   should have a `CELERY_` prefix but remember to use only the new settings names
```

Comment:
I don't think we need this. There is higher probability that it becomes obsolete than somebody will make use of it.